backend/controller/controller.go

```diff
@@ -1828,12 +1823,16 @@ func (s *Service) syncRoutesAndSchema(ctx context.Context) (ret time.Duration, e
 		}
```

Comment:
Maybe rename this function to syncState()?

Reply:
These are still technically routes, if we end up syncing more stuff then I think we should rename it.

Reply:
Hehe good point :)